packages/gatsby/src/commands/build.ts

```diff
@@ -259,6 +259,7 @@ module.exports = async function build(program: IBuildArgs): Promise<void> {
             type: `LOG_ACTION`,
             action: {
               type: `ENGINES_READY`,
+              timestamp: new Date().toJSON(),
```

Comment:
At least our tests exepct all `LOG_ACTION` messages to contain `timestamp`. Decided to add it here instead of filtering it out in tests as this timestamp will be harmless.